packages/fluentui/projects-test/assets/cra/App.tsx

```diff
@@ -26,11 +26,14 @@ class App extends React.Component {
             <CalendarIcon circular bordered />
           </Animation>
           <Attachment header="Document.docx" />
-          <Avatar image="//placehold.it" />
+          <Avatar image="https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/large/ade.jpg" />
```

Comment:
is our CDN really better than `//placehold.it`?

I wonder if it's better to just fix the by using correct placehold url such as http://placehold.it/100x100

Reply:
I'm not sure what the SLA is for placehold.it (if they even have one), so I actually feel better about using our CDN here--it's a standard Azure CDN which should be quite reliable.